### PR TITLE
[MIG] web_duplicate_visibility: Migration to 10.0

### PR DESCRIPTION
**Pull request by yousef-soliman**
Just had to change the XPath for QUnit tests javascript file.

### CODE REVIEW

**Comment by yousef-soliman** [comment]
ping @pedrobaeza

**Comment by pedrobaeza** [comment]
Sorry, but you have to get reviewers to this. Try to review others' PRs and ask in exchange to review yours.

File: web_duplicate_visibility/views/assets.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <template id="qunit_suite"
               name="web_duplicate_visibility qunit assets"
               inherit_id="web.qunit_suite">
-        <xpath expr="//html/head"
+        <xpath expr="//script[last()]"
```
Comment (JayVora-SerpentCS) [inline web_duplicate_visibility/views/assets.xml:17]: The code is right. Did you check on runbot?

File: web_duplicate_visibility/__manifest__.py [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
     "website": "https://odoo-community.org/",
     "author": "Pierre Verkest <pverkest@anybox.fr>,"
               " Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
-    'installable': False,
+    'installable': True,
     "depends": [
         "base",
         "web",
```
Comment (JayVora-SerpentCS) [inline web_duplicate_visibility/__manifest__.py:17]: Keep only 'web'.

**Comment by JayVora-SerpentCS** [comment]
@yousef-soliman Add your name /email in the contributor section of readme of the module

File: web_duplicate_visibility/views/assets.xml (hunk 1 above)
Comment (yousef-soliman) [reply web_duplicate_visibility/views/assets.xml:17]: @JayVora-SerpentCS  first thanks for review :),
yes i did it was working well

**Pull request closed without merging**